src/client/testing/fixtures.ts

```diff
@@ -4,12 +4,28 @@ process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
 import { test as baseTest } from '@playwright/test';
 
 export * from '@playwright/test';
-export const test = baseTest.extend<object, { newUserName: string }>({
+export const test = baseTest.extend<
+  object,
+  { newUserName: string; externalUserName: string }
+>({
   newUserName: [
     async ({}, use) => {
       const userName = `int-test-user-${test.info().workerIndex}@nhs.net`;
       await use(userName);
     },
     { scope: 'worker' },
   ],
+  externalUserName: [
+    async ({}, use) => {
+      const nonNHSuserName = `external-user-${test.info().workerIndex}@gmail.com`;
+      await use(nonNHSuserName);
+    },
+    { scope: 'worker' },
+  ],
 });
+
+export const testuser8_emailId = `zzz_test_user_8@nhs.net`;
+export const testuser9_emailId = `zzz_test_user_9@nhs.net`;
+export const testuser10_emailId = `zzz_test_user_10@nhs.net`;
+export const testuser11_emailId = `zzz_test_user_11@nhs.net`;
```

Comment:
I think it'd be clearer to store this information alongside the usernames/passwords. These are currently declared in the .env file and piped through as config.

If we move the sites out into a new seed-data folder as I've suggested below, we could also move the users data out there too. The existing pattern of piping them through from env doesn't really work anymore now the data we need is getting larger